CASSANDRA.md

```diff
@@ -153,9 +148,19 @@ object Phantom extends App {
 }
 ```
 
-Phantom requires mapping classes to lift the database model to DSL types. The query definition also requires special equality operators.
+Phantom requires mapping classes to lift the database model to DSL types. The query definition also requires special equality operators. The database model however adds fully type-safe schema compatible operations at compile time, meaning
+phantom will not allow you to produce invalid queries with respect to the Cassandra schema, but it relies on
+using phantom to produce the schema in the first place.
+
+As opposed to the Java Driver, phantom is schema aware, whereas the other two tools simply generate
+the correct CQL string with respect to the Scala type, but not with respect to the schema.
+
+Quill also has a feature that allows checking query validity against the pre-existing Cassandra schema, but assumes
+the schema has already been created externally. Phantom expects to also manage schema creation for you, and as part of the
```

Comment:
Let's compare only the FOSS projects. Could you remove the last phrase (`Phantom expects...`)?